assets/js/study.js

```diff
@@ -642,6 +666,27 @@ async function setup_progress_reset() {
     }
 }
 
+/**
+ * Initiate all option values on the page.
+ */
+function set_options_values() {
+    let move_delay = document.getElementById("move_delay_time");
+    move_delay.innerHTML = move_delay_time;
```

Comment:
Can you `innerHTML` -> `innerText`? I know we control the input but I still would feel better with `innerText` :)

Reply:
Absolutely. There it is.